### PR TITLE
README.md - include finalConfigure() instructions

### PR DESCRIPTION
**Pull request by advayDev1**

### CODE REVIEW

**Comment by confile** [comment]
LGTM. 

Isn't there a more elegant solution than forcing the user to call a function in the extension block?

File: README.md [hunk 1]
```diff
@@ -37,6 +37,11 @@ Tracked issue: https://github.com/j2objc-contrib/j2objc-gradle/issues/130**
         xcodeProjectDir "${projectDir}/../ios"   // Xcode workspace directory
         xcodeTarget "MyTarget"                   // Xcode application target name
         // More: https://github.com/j2objc-contrib/j2objc-gradle/blob/master/src/main/groovy/com/github/j2objccontrib/j2objcgradle/J2objcPluginExtension.groovy#L25
+        
+        // Must always be the last statement in the j2objcConfig block.
+        // You must include this call even if you do not want to configure
```
Comment (brunobowden) [inline README.md]: I'd combine the last two lines of comments in to a single line:

       // You must include this call even if you do not configure any other settings

**Comment by brunobowden** [comment]
LGTM - see single comment

File: README.md (hunk 1 above)
Comment (advayDev1) [inline README.md]: added, slightly different wording to ensure users keep it at the end.

**Comment by advayDev1** [comment]
@confile, not at the moment, see #158 comment:
"The inability to reconfigure a previously
created native component is an issue others are hitting too:
https://discuss.gradle.org/t/problem-with-model-block-when-switching-from-2-2-1-to-2-4/9937"
:(

**Pull request merged**